src/c4/QueryEnv.hh

```diff
@@ -0,0 +1,117 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   ds++/QueryEnv.hh
+ * \author Tim Kelley
+ * \date   Fri Jun 7 08:06:53 2019
+ * \brief  Functions for working with your environment
+ * \note   Copyright (C) 2019 Triad National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef Query_Env_hh
+#define Query_Env_hh
+
+#include <cstdlib> // getenv, setenv
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <type_traits>
+#include <utility> // pair
+
```

Comment:
I think you need to add `#include <tuple>`.

Reply:
Done, thanks.